### ISSUE TITLE
Same contextID for multipls tags can't be controlled via dlt-control command 

### ISSUE THREAD

**Issue by MAZEEM2** [COVESA/dlt-daemon#709]
In json file i have added the different keys for same tag.
{
  "NAV-TomtomMessageProcessor": {
    "tag": "NAV1",
	"description": "firstTag"
  },
  "NAV-NavProviderInitiator": {
    "tag": "NAV1",
	"description": "SecondTag"
  },
  "c.t.n.s.c.NavAppClientImpl": {
    "tag": "NAV1",
	"description": "ThirdTag"
  }, 
  "ServiceIntentProvider": {
    "tag": "NAV1",
	"description": "FourthTag"
  }, 
  "ClientSDK": {
    "tag": "NAV1",
	"description": "FifthTag"
  },
  "c.t.n.s.a.c.d": {
    "tag": "NAV1",
	"description": "SixthTag"
  },
}

And in dlt-logd-converter.cpp i have modified the values which will get read from the json.
string json_ctxID = (*iter)["tag"].asString();
string json_tag = iter.key().asString();

basically i swapped the key with tag and read it like this so that for different tags  the context id should be same .
As of now in dlt-daemon 2.18.10 its only one-to-one mapping of tag with context id present
In my code i want to make it one-to-many means one contextid with many tags.
After doing this above changes i am able to see the context id is coming as NAV1 for all the tags in the dlt-viewer  but for controlling its only controlling the single tag.

**Comment by AnandShastry** [comment]
better approach, please try this
json_ctxID = iter->get("tag", "").asString();
json_tag = iter.name();

**Comment by minminlittleshrimp** [comment]
Also, the dlt-logd-converter needs some refactoring due to not using the libjson APIs at all, but due to some dummy things with unordered_map. This was developed long time ago based on logcat AOSP.
